calebwagner/cars-r-us
# added all user option modules

I added modules Interior.js, Technology.js, Wheels.js.
Steps to check code:
1. ensure that modules appear in Cars-R-Us.js module
2. ensure that all list items appear in browser
